agenta-backend/agenta_backend/migrations/20240110001454_initial_migration.py

```diff
@@ -0,0 +1,123 @@
+from agenta_backend.models.db_models import (
+    AppDB,
+    OrganizationDB,
+    UserDB,
+    TestSetDB,
+    EvaluationDB,
+    EvaluationScenarioDB,
+    EvaluationScenarioInputDB,
+    EvaluationScenarioOutputDB,
+    EvaluatorConfigDB,
+    HumanEvaluationDB,
+    HumanEvaluationScenarioDB,
+    HumanEvaluationScenarioInput,
+    HumanEvaluationScenarioOutput,
+    OldEvaluationDB,
+    OldEvaluationScenarioDB,
+)
+
+from beanie import free_fall_migration, Link
+
+
+class Forward:
+    @free_fall_migration(document_models=[OldEvaluationDB, EvaluationDB])
+    async def move_old_evals_to_new_evals_document(self, session):
+        async for old_eval in OldEvaluationDB.find_all():
+            eval_config = EvaluatorConfigDB(
```

Comment:
@aybruhm I think this would result in one new evaluator for each evaluation! Is there any better solution for this?